app/src/main/java/com/xffffff/wellfed/common/DBAdapter.java

```diff
@@ -49,8 +49,7 @@ public interface OnDataChangedListener {
      * @param query the query
      */
     public DBAdapter(Query query) {
-        this.mQuery = query;
-        startListening();
+        this.setQuery(query);
```

Comment:
may i know why was this required?

Reply:
It's not, but I was just trying to make it the same to figure out why it was bugging out earlier. I saw that it is cleaner, so I left it.

Reply:
sounds good